<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titlelabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ff0000;&quot;&gt;Nguyễn Huỳnh Yến Nhi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowPNGpushButton">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>120</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show .PNG format</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowGIFpushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show .Gif format</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagelabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>541</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image/picture.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="FontsizepushButton">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>72</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size ++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AlignLeftPushButton">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>72</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Align Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangetextpushButton">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>72</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AlignRightPushButton">
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>72</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Align Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AlignCenterPushButton">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>72</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Align Center</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
